### PR TITLE
prepare glossary for publish

### PR DESCRIPTION
**Pull request by phrawzty**
### What does this PR do?

This (re-)adds the Glossary section. It is not complete, but it is a good start and we can iterate from here.

### Motivation

Improve the developer experience by making things easier to find and learn about.

### Preview link

https://docs-staging.datadoghq.com/phrawzty/prep_glossary_for_publish/glossary

### Additional Notes

This does *not* add the Glossary to the navigation menu. We must decide where we'd like this nav item to be positioned. I'm open to discussion on this. 😄 
*Edit*: It already had a nav bar config, so I uncommented it.

### CODE REVIEW

**Comment by l0k0ms** [comment]
plop, why are you using H4 for the titles? are your going to nest those words under an other H3/H2 section ?

**Comment by phrawzty** [comment]
@l0k0ms No particular reason, I suppose. I like the way `####` is rendered on the docs site. What would you recommend?

**Comment by l0k0ms** [comment]
whatever really :) I do like it this way too. my only concern here is that the table of content only pics up H2/H3 you can disable it with `disable_toc: true` as a param

File: content/glossary.md [hunk 1]
```diff
@@ -1,72 +1,52 @@
...
 See the [DogStatsD documentation][4] for more information.
 
-**Downtime**
-
-**Embed**
-
-**Integration**
+#### Integration
 
 An Integration is a way to get data from your systems into Datadog. Integrations gather data from a given source, ensure that those data are classified correctly, and provide some other assets to assist with configuration and usage. Data sources can be pretty much anything from daemons on a server, to cloud services, to third-party APIs, and more.
```
Comment (irabinovitch) [inline content/glossary.md]: s/pretty much//

**Pull request merged**